core/vm/evm.go

```diff
@@ -469,14 +518,25 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 	// Ensure there's no existing contract already at the designated address
 	contractHash := evm.StateDB.GetCodeHash(address)
 	if evm.StateDB.GetNonce(address) != 0 || (contractHash != (common.Hash{}) && contractHash != emptyCodeHash) {
+		// TODO this is a case where the witness should touch the already-created account
+		// however, it seems impossible to ever actually encounter this.
```

Comment:
I don't quite see why this is a TODO then?

Reply:
I've removed it.